### PR TITLE
:book: Add navbar on top of homepage

### LINKED ISSUE #1239: :book: Add a link to the blog on the Kairos homepage
Right now a user landing on the home page needs to click "documentation" before they can see there's also a blog. This is not very intuitive

### PR DESCRIPTION
**Pull request by mauromorales**
:warning: Please check to see if you agree with the look, including on mobile

Additionally, this PR removes the use of having the static asset index.css in favor of using Hugo's system to extend styles. This is better because if index.css was cached, it didn't include a fingerprint and therefore, if there were changes, a user would not trigger a refresh and see the old styles.

Fixes #1239

### CODE REVIEW

**Comment by mudler** [comment]
just few points:
- the search seems broken
- the logo on the top left seems a repetition, maybe we shouldn't include it
- Maybe we should have a "Star us on github?" and a Blog link only, there are already call for actions buttons like Documentation and Download in the center

**Comment by mauromorales** [comment]
@mudler thanks for the feedback, pls give it another look

**Comment by mudler** [comment]
looking good! don't hate me now :see_no_evil: .. maybe we can also keep the Community link now?

**Review by mudler — APPROVED** [review]
nais!

File: docs/layouts/index.html [hunk 1]
```diff
@@ -1,24 +1,32 @@
...
+            <li class="nav-item mr-4 mb-2 mb-lg-0">
+              <a class="nav-link" href="/blog/"><span>Blog</span></a>
+            </li>
+            <li class="nav-item mr-4 mb-2 mb-lg-0 mt-lg-2">
+              <a class="github-button" href="https://github.com/kairos-io/kairos" data-size="large" data-show-count="true" aria-label="Star kairos-io/kairos on GitHub">Star</a>
+            </li>
+          </ul>
+        </div>
+        <div class="navbar-nav d-none d-lg-block">
+          {{ partial "search-input.html" . }}
```
[the remark this answers is not in the document]
Comment (mauromorales) [reply docs/layouts/index.html:27]: indeed, this brings in lunr.js which is used for the search, plus other scripts for docsy

**Comment by mauromorales** [comment]
> looking good! don't hate me now see_no_evil .. maybe we can also keep the Community link now?

no worries, happy to do as many iterations as needed ... added the community link and I think it looks better with it

**Review by mudler — APPROVED** [review]
Looking great! Also checked on mobile, all good!

**Pull request merged**